man/systemd-nspawn.xml

```diff
@@ -183,9 +183,16 @@
         <listitem><para>If specified, the container is run with a temporary snapshot of its file system that is removed
         immediately when the container terminates. May not be specified together with
         <option>--template=</option>.</para>
-        <para>Note that this switch leaves host name, machine ID and
-        all other settings that could identify the instance
-        unmodified.</para></listitem>
+        <para>Note that this switch leaves host name, machine ID and all other settings that could identify the
+        instance unmodified. Please note that — as with <option>--template=</option> — taking the temporary snapshot is
+        more efficient on file systems that support subvolume snapshots or 'reflinks' naively (<literal>btrfs</literal>
```

Comment:
```suggestion
        more efficient on file systems that support subvolume snapshots or 'reflinks' natively (<literal>btrfs</literal>
```

Reply:
Forgot to fix this before the merge, but #11863 finally fixes this one too!